doc/whats_new.rst

```diff
@@ -58,6 +60,8 @@ API
 
     - :func:`mne.io.read_raw_brainvision` now has ``event_id`` argument to assign non-standard trigger events to a trigger value by `Teon Brooks`_
 
+    - :func:`mne.read_epochs` now has ``add_eeg_ref=False`` by default, since average EEG reference can be added before writing or after reading using the method :func:`mne.Epochs.add_eeg_ref`.
+
```

Comment:
@agramfort from looking at the code, it really seems like the cleanest solution is to eliminate the `add_eeg_ref` arguments to our constructors altogether. It's the most explicit about what's happening to data. Maybe target for 0.12 so we can test thoroughly?